Terminal.Gui/Drivers/CursesDriver.cs

```diff
@@ -266,6 +266,7 @@ public override void Init (Action terminalResized)
 				Colors.Menu.Focus = Curses.A_BOLD | MakeColor (Curses.COLOR_WHITE, Curses.COLOR_BLACK);
 				Colors.Menu.HotNormal = Curses.A_BOLD | MakeColor (Curses.COLOR_YELLOW, Curses.COLOR_CYAN);
 				Colors.Menu.Normal = Curses.A_BOLD | MakeColor (Curses.COLOR_WHITE, Curses.COLOR_CYAN);
+				Colors.Menu.Disabled = MakeColor(Curses.COLOR_WHITE, Curses.COLOR_CYAN);
 
```

Comment:
While this is fine, it does not handle the black and white setting for Colors.Menu.Disabled, so you need to figure out what color you would like to use for that and set it there.